tests/test_operation.py

```diff
@@ -96,118 +99,125 @@ def test_heisenberg(self, cls, tol):
             U_high_order = np.array([U] * 3)
             op.heisenberg_expand(U_high_order, len(op.wires))
```

Comment:
Maybe I'm missing it, but are there any changes in this test, or is the only change `cls`->`test_class`?